renpy/exports.py

```diff
@@ -4281,13 +4287,25 @@ class LastSay():
     Returned by the last_say function.
     """
 
+    def __init__(self, who, what, args, kwargs):
+        self._who = who
+        self.what = what
+        self.args = args
+        self.kwargs = kwargs
```

Comment:
I _think_ you can do this, and at which point `last_say` could simply be an alias to `LastSay`, and since unpickling doesn't go through `__init__` it shouldn't impact loading.
```suggestion
    def __init__(self):
        self._who = renpy.store._last_say_who
        self.what = renpy.store._last_say_what
        self.args = renpy.store._last_say_args
        self.kwargs = renpy.store._last_say_kwargs
```

It may not be worth the trouble though, so I'll leave it to you as an option. 🙂

Reply:
I would do it if it were only me, but I'd rather keep the code vaguely similar to how Lez wrote it. And it could allow creating custom LastSay entries, in general that's a thing I like to be able to do when creating classes.

Reply:
Fair. In that case I do think it's worth calling `__init__` with positional args from `last_say()`. Since it's a private interface we don't need to worry about changing how we call it in future so they're not really providing any benefit, and arbitrarily passing them as keyword args seems a bit - for lack of a better descriptor - odd. 😛